scripts/script.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const container = document.querySelector('.container');
+
+function gettingInfo(url) {
+  return fetch(url).then(response => response.json());
+}
+
+async function creatingPostsLists() {
+  const basicUrl = 'https://jsonplaceholder.typicode.com/';
+  const [postsList, usersList, commentsList] = await Promise.all([
+    gettingInfo(`${basicUrl}posts`),
+    gettingInfo(`${basicUrl}users`),
+    gettingInfo(`${basicUrl}comments`)]);
+    
+  const basicClassList = ['ui', 'grid', 'centered'];
+
+  
+  for (const posts of postsList) {
+    const divBox = document.createElement('div');
+    divBox.classList.add(...basicClassList);
+
+    const postsTitle = posts.title;
+    const postsBody = posts.body;
+    const userName = usersList.find(user => user.id === posts.userId).name;
+
+    const postComments = commentsList.filter(post => post.postId === posts.id);
```

Comment:
```suggestion
    const postComments = commentsList.filter(comment => comment.postId === posts.id);
```

ти перебераєш масив з коментарями, а не постами

Reply:
називай змінні відповідно, а то потім зрозуміти що де є дуже складно